File: app/src/main/java/com/example/msdc/api/SeasonResponse.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class SeasonResponse {
    @SerializedName("_id")
    String id;

    @SerializedName("episodes")
    ArrayList<EpisodeResult> episodeResults;

    public String getId() {
        return id;
    }

    public ArrayList<EpisodeResult> getEpisodeResults() {
        return episodeResults;
    }
}

File: app/src/main/java/com/example/msdc/api/SeasonResponse.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class SeasonResponse (
    @field:SerializedName("_id") val id: String? = null,
    @field:SerializedName("episodes") val episodeResults: ArrayList<EpisodeResult>? = null
) : Parcelable